nats/README.md

```diff
@@ -17,7 +17,7 @@ This demo shows how to stream changes from MySQL database running on a local mac
 We will start a pre-populated MySQL database that is the same as used by the Debezium [tutorial](http://debezium.io/docs/tutorial/):
 
 ```
-docker run -it --rm --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=debezium -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw debezium/example-mysql:0.9
+docker run -it --rm --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=debezium -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw debezium/example-mysql:0.10
```

Comment:
@gunnarmorling Why `0.10` is used here and `0.9` in other cases?

Reply:
This was an oversight, good catch. Updated to 0.9.